06a-WGS.Rmd

```diff
@@ -0,0 +1,118 @@
+- [Samtools](http://www.htslib.org/) including tools for converting fastq to BAM files while mapping genes to the genome, duplicate read marking, and sorting reads
+- [Picard2](https://broadinstitute.github.io/picard/) including tools to covert fastq to SAM files, filter files, create indices, mark read duplicates, sort files, and merge files
+- [GATK](https://gatk.broadinstitute.org/hc/en-us) is a comprehensive set of tools from the Broad Institute for analyzing many types of sequencing data. For pre-processing, the print read function is very beneficial for writing the reads from a BAM or SAM file that pass specific criteria to a new file
+
+These are the common tools used for somatic and germline variant identification. Note many researchers will often use a combination of these tools to narrow down only variants that are identified using a combination of these analysis algorithms:
+- [Mutect2](https://gatk.broadinstitute.org/hc/en-us/articles/9570422171291-Mutect2) This is a beneficial variant calling tool with functions including using a "panel of normals" to better compare disease samples to normal and filtering functions for samples with orientation bias artifacts (FFPE samples) called F1R2
+- [Varscan 2](https://varscan.sourceforge.net/) This is a helpful tool that utilizes a heuristic/statistic approach to variant calling. It is generally used and has capabilities for calling multi-sample variants. It can also be used for identifying copy number alterations in tumor-normal pairs
+- [MuSE](https://bioinformatics.mdanderson.org/public-software/muse/) This is a beneficial mutation calling tool when you have both tumor and normal datasets. It uses a Markov Substitution model for Evolution, which models the evolution of the allelic composition of the tumor and normal tissue at each reference base. 
+- [SvABA](https://github.com/walaj/svaba) This tool is especially useful for calling insertions and deletions (indels) because it utilizes a String Graph Assembler to assemble aberrantly aligned sequence reads that reflect indels or structural variants. Indels can be difficult to detect with standard alignment-based variant callers. 
+- [Strelka](https://github.com/Illumina/strelka) This is a small variant caller designed by illumina. It is used for identifying germline variants in cohorts of samples and somatic variants in tumor/normal sample pairs. 
+- [SomaticSniper](https://gmt.genome.wustl.edu/packages/somatic-sniper/) SomaticSniper can be used to identify SNPs in tumor/normal pairs. It calculates the probability that the tumor and normal genotypes are different and reports this probability as a somatic score. 
+- [Pindel](https://www.sanger.ac.uk/tool/pindel/) Pindel is a tool that uses a pattern growth approach to detect breakpoints of large deletions, medium size insertsion/inversion, tandem duplications. 
```

Comment:
We will want to make sure to feature Lancet: https://github.com/nygenome/lancet Not only is it a pretty decent caller [with a nice paper that compares its methods to these other callers](https://www.nature.com/articles/s42003-018-0023-9) , but it's also ITCR funded I believe.

Reply:
Sorry I took this one off - I couldn't find the information about it anywhere - thanks for adding the link!

Reply:
Selfishly I also kinda want to plug this snv caller analysis I've done where we compared these callers -- the code is all open source so folks can borrow any parts of it: https://github.com/AlexsLemonade/OpenPBTA-analysis/tree/master/analyses/snv-callers